academy/models/teachers.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models, fields
+
+
+class Teachers(models.Model):
```

Comment:
classes always needs to be declared as singular

```suggestion
class Teacher(models.Model):
```